valtimo-implementation/process-blueprints.md

```diff
@@ -0,0 +1,176 @@
+|   |   |   |   |-- resources
+|   |   |   |       |-- bpmn
+|   |   |   |       |-- dmn
+|   |   |   |       |-- config
+|   |   |   |           |-- document
+|   |   |   |           |-- form
+|   |   |   |           |-- form-flow
+|   |   |   |           |-- form-link
+|   |   |   |           |-- liquibase
+|   |   |   |           |-- proces-document-link
+|   |   |   |           |-- application.yml
+|   |   |   |-- build.gradle/pom.xml
+|   |   |-- frontend
+|   |   |   |-- app
+|   |   |   |-- assets
+|   |   |   |-- environments
+|   |   |   |-- package.json
+|   |   |-- README.md
+|   |-- ...
+|-- CONTRIBUTING.md
+|-- README.md
+```
+
+Almost all the files in `backend` can be copied into the back-end of an implementation project. The exceptions are:
+* `application.yml`. This only includes the configurations required by the process blueprint. These should be merged
+with the `application.yml` you already have.
+* `build.gradle` or `pom.xml`. This only includes the dependencies required by the process blueprint. These should be
+merged with the `build.gradle` or `pom.xml` you already have.
+
+Almost all files in `frontend` can be copied into the front-end of an implementation project. The exceptions are:
+*`package.json`. This only includes the dependencies required by the process blueprint. These should be merged with
+the `package.json` you already have.
+
+### Configuring and customizing the process blueprint
+
+A process blueprint will not work out of the box. Plugins might need to be configured, and process links need to be set.
+Each process blueprint comes with a README that explains the specifics that should be configured in order for the
+process blueprint to be used. In addition to this, consider the following checklist:
+
+* Check each task in the BPMN definition. Are all tasks defined correctly?
+* If a task relies on a method for a specific spring bean, are both the bean and the method present?
+* Does this process blueprint rely on plugins? If so, these will have to be configured, and process links will have to
+be configured for the appropriate tasks as well.
+* Does every user task have a form or form flow associated with it?
+* If there are form flows, are all the forms it references present?
+
+As always, be sure to follow the specific instructions that are included with the process blueprint.
+
+## Creating process blueprints for GZAC
+It is encouraged to share processes that can potentially be used by other municipalities. Another municipality can then
+implement the same process with minimal effort.
+
+### Privacy and intellectual property
+Sharing these processes is done through a public repository. This has some effects that a publisher should know before
+you continue.
+
+Shared resources should not contain:
+* Any privacy- or security-sensitive information like:
+    * Names, addresses or other data of real people.
+    * Credentials of users or services
+* The municipality name in code, labels, etc.
```

Comment:
municipality -> organization ? (I'm not sure, but I could imagine a non-municipality organization using a municipality process